app/services/author_notifier_service.rb

```diff
@@ -18,6 +18,13 @@ def notify!
         edition_admin_url,
         public_document_url,
       ).deliver_now
+    rescue Notifications::Client::BadRequestError => e
+      if !%w[integration staging].include?(ENV["SENTRY_CURRENT_ENV"]) || e.message !~ /team-only API key/
```

Comment:
I don't think this logic is quite correct (at least for Whitehall) as in integration `SENTRY_CURRENT_ENV` equals `"integration-blue-aws"`. For staging it is fine though. 👍 

Side note: we could use `exclude?` here instead of inverting `includes?` boolean.

Reply:
thanks for noticing that environment @1pretz1!

Reply:
Elsewhere we use `ENV.fetch("GOVUK_WEBSITE_ROOT") =~ /integration|staging/`; I think for this we'd still want to check the sentry environment but we could still do `=~` instead of `include?`.

Reply:
How about we instead check it's production
`raise if ENV["SENTRY_CURRENT_ENV"] =~ /production/`?

Reply:
I think @thomasleese suggested doing it this way around because if `SENTRY_CURRENT_ENV` is unset we'd want it to default to the production behaviour.